examples/cookbook/design/fonts/pubspec.yaml

```diff
@@ -0,0 +1,89 @@
+name: fonts
```

Comment:
Feel free to rip all the autogenerated comments out of this file.

Reply:
Done!

Reply:
Took out all of the autogenerated comments in the other files as well.